apps/transport/test/transport/gbfs_metadata_test.exs

```diff
@@ -662,6 +663,140 @@ defmodule Transport.GBFSMetadataTest do
     end
   end
 
+  describe "types" do
+    test "vehicles and stations feeds" do
+      setup_response(
+        vehicle_status_url = "https://example.com/vehicle_status",
+        Jason.encode!(%{data: %{vehicles: [%{bike_id: Ecto.UUID.generate()}]}})
```

Comment:
Rajouter un commentaire ici ou au niveau de l’assertion pour dire "the bike has no docking station information, and is then considered as free floating".